docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  api:
+    build:
+      dockerfile: Dockerfile
+      context: .
+      target: development
+    volumes:
+      - .:/usr/app
+    env_file:
+    - .env
+    command: npm run start:dev
+    ports:
+      - 3000:3000
+    depends_on:
+      - postgres
+  pgadmin:
```

Comment:
to jest bez sensu bo lokalnie sie mozesz laczyc z baza nie musisz tego wystawiac

Reply:
Niby tak, ale wystawimy pgadmin na docker to można się połączyć przez przeglądarkę :)

Reply:
to wspolczuje kazdemu kto bazy danych z przegladarki obsluguje ;) juz o wiele lepiej przez IDE sie polaczyc

Reply:
czyli co mam tu konkretnie zrobić? bo już nie wiem

Reply:
chodzi o to aby wyrzucić pgadmin z docker-compose
będziemy się łączyć z bazą z aplikacji pgadmin albo innej apki obsługującej postgres